lib/taurus/qt/qtgui/taurusgui/res/setup.template

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+"""
+This file was autogenerated by taurusgui --new-gui.
+"""
+
+from setuptools import setup, find_packages
+
+# version (bumpversion style)
```

Comment:
> I think is the developer who has to take the decision to use bumpversion and how to use it.

The user could just ignore the .bumpversion file and work without it... but if we provide it we are encouraging/facilitating a good practice

>  generated project are not under a git repo

that is not relevant. bumpversion can also be used without git

> I would not do more project configuration.

That's fine by me. It was only a suggestion